src/Stratis.Bitcoin.IntegrationTests.Common/Runners/StratisBitcoinPowRunner.cs

```diff
@@ -23,17 +23,21 @@ public override void BuildNode()
         {
             var settings = new NodeSettings(this.Network, args: new string[] { "-conf=bitcoin.conf", "-datadir=" + this.DataFolder });
 
-            this.FullNode = (FullNode)new FullNodeBuilder()
-                .UseNodeSettings(settings)
-                .UseBlockStore()
-                .UsePowConsensus()
-                .UseMempool()
-                .AddMining()
-                .UseWallet()
-                .AddRPC()
-                .UseApi()
-                .MockIBD()
-                .Build();
+            var builder = new FullNodeBuilder()
+                        .UseNodeSettings(settings)
+                        .UseBlockStore()
+                        .UsePowConsensus()
+                        .UseMempool()
+                        .AddMining()
+                        .UseWallet()
+                        .AddRPC()
+                        .UseApi()
+                        .MockIBD();
+
+            if (this.ServiceToOverride != null)
```

Comment:
Don't use block store use base feature instead else its confusing, it does nto matter so much because the services DI is shared with all features